app/main/forms/frameworks.py

```diff
@@ -76,3 +76,48 @@ def __init__(self, lot_name, *args, **kwargs):
                                                 "Roles won’t be copied if they have new questions."
         else:
             self.copy_service.question_advice = "You still have to review your service and answer any new questions."
+
+
+class LegalAuthorityForm(FlaskForm):
+    OPTIONS = [
+        {
+            "value": "yes",
+            "label": "Yes",
+        },
+        {
+            "value": "no",
+            "label": "No",
+        },
+    ]
+    legal_authority = DMRadioField(
+        "Do you have the legal authority to sign on behalf of your company?",
+        hint="For example, you are a director or company secretary.",
+        validators=[InputRequired(message="Select yes if you have the legal authority"
+                                          " to sign on behalf of your company")],
+        options=OPTIONS)
+
+
+class SignFrameworkAgreementForm(FlaskForm):
+
+    # Intended use of camel case here to match expected API fields
+    signerName = DMStripWhitespaceStringField(
+        "Your full name",
+        validators=[
+            DataRequired(message="You must provide your full name."),
+            Length(max=255, message="You must provide a name under 256 characters."),
```

Comment:
I think the [guidance](https://design-system.service.gov.uk/components/error-message/) now is to avoid the word 'must' - instead use an instruction eg 'Enter a name under 256 characters'. (cc @NoraGDS)

Reply:
Yes - on the error message not using 'must' and not sure max length is applicable for this question

•Enter your full name
•Enter your first and last name

Reply:
Thanks, I've updated these error messages now. @NoraGDS max length is a technical constraint of our database.